packages/app/src/app/overmind/namespaces/workspace/actions.ts

```diff
@@ -106,15 +106,31 @@ export const sandboxInfoUpdated: AsyncAction = withOwnedSandbox(
 
       effects.analytics.track(`Sandbox - Update ${event}`);
 
-      sandbox.title = project.title;
-      sandbox.description = project.description;
-      sandbox.alias = project.alias;
-
-      const updatedSandbox = await effects.api.updateSandbox(sandbox.id, {
-        title: project.title,
-        description: project.description,
-        alias: project.alias,
-      });
+      const payload: {
+        title?: string;
+        description?: string;
+        alias?: string;
+      } = {};
+
+      if (hasChangedTitle) {
+        payload.title = project.title;
+        sandbox.title = project.title;
+      }
+
+      if (hasChangedDescription) {
+        payload.description = project.description;
+        sandbox.description = project.description;
+      }
+
+      if (hasChangedAlias) {
+        payload.alias = project.alias;
+        sandbox.alias = project.alias;
+      }
+
+      const updatedSandbox = await effects.api.updateSandbox(
+        sandbox.id,
+        payload
```

Comment:
Hm, could you not just update the sandbox with the project? Then update the sandbox? Does not matter if it has changed or not?

Reply:
So you mean sending the sandbox as payload?